src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerFullGroupTable.class.php

```diff
@@ -54,15 +54,10 @@ public function Init(&$oTableConf)
             $cachedTableObj = $this->getTableFromSessionCache($listCacheKey);
         }
 
-        $aOrderData = array();
-        if (null !== $cachedTableObj) {
-            $aOrderData = $cachedTableObj->orderList;
```

Comment:
Couldn't the $cachedTableObj contain something?

Reply:
It can, but `$aOrderData` is only ever used in the statement below, that has `if (null === $cachedTableObj)` wrapped around it - which means it is only ever used in it's initial, empty array state.

Reply:
Ah, I see: it is properly used in the else case that follows.